.github/workflows/event-merge_queue.yml

```diff
@@ -0,0 +1,38 @@
+name: Merge pull request
+
+on:
+  merge_group:
+    types: [checks_requested]
```

Comment:
As far as we understood, this flow defines what checks should pass for a PR to be merged. This will replace defining some flows as “required” before pressing “merge”, and instead of labeling (which is kind of a manual way to do the same thing)